### PR TITLE
More pedantic ESlint setup

### PR DESCRIPTION
**Pull request by kirushik**

### CODE REVIEW

File: src/util/bipPhrase.js [hunk 1]
```diff
@@ -1,37 +1,2067 @@
...
+    "universe",
+    "unknown",
+    "unlock",
+    "until",
+    "unusual",
+    "unveil",
+    "update",
+    "upgrade",
+    "uphold",
+    "upon 100 13116  100 13116    0     0  45660      0 --:--:-- --:--:-- --:--:-- 45700",
```
Comment (danforbes) [inline src/util/bipPhrase.js:1913]: What's this?

Comment (kirushik) [reply src/util/bipPhrase.js:1913]: Oh crap; that's exactly a type of thing I was worrying about. Thanks for noticing!

Comment (danforbes) [reply src/util/bipPhrase.js:1913]: Hooray for PRs!

Comment (danforbes) [reply src/util/bipPhrase.js:1913]: How did this happen though?

Comment (kirushik) [reply src/util/bipPhrase.js]: I used `:r! curl 'https://raw.githubusercontent.com/bitcoin/bips/master/bip-0039/english.txt'` to fetch the bip dictionary, and apparently some curl's status has been captured as well.
This list also missed the first word.

It's not an issue to alter this dictionary, though -- it's only used one-way to generate passphrases when creating shards.
Those phrases are just treated as user-input stings in all other places, so fixing the wordlist won't break anything.

Comment (danforbes) [reply src/util/bipPhrase.js]: Ahhhh gotcha

Comment (danforbes) [reply src/util/bipPhrase.js]: Why are you changing the word list as part of this PR? Can you just format the existing one or is there something about the existing list that the linter doesn't like?

Comment (kirushik) [reply src/util/bipPhrase.js]: > Why are you changing the word list as part of this PR? Can you just format the existing one or is there something about the existing list that the linter doesn't like?

I'm just formatting existing one.
The typo was there in the list all along, and the new formatting just made it obvious (which is the point of sensible formatting policy, I guess.)

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+module.exports = {
```
Comment (Stefie) [inline .eslintrc.js:1]: Jaco has this eslint & tslint config that he uses in all of the PolkadotJS packages and we're also using it in quite a few of the Parity JS projects, so I'd say it's battle tested.
Might be worth having a look even if you don't want to 100% copy it:
https://github.com/polkadot-js/dev/blob/master/packages/dev/config/eslint.js
https://github.com/polkadot-js/dev/blob/master/packages/dev/config/tsconfig.json

File: jest.config.js [hunk 3]
```diff
@@ -1,24 +1,18 @@
 module.exports = {
```
Comment (Stefie) [inline jest.config.js:1]: The PolkadotJS jest config:
https://github.com/polkadot-js/dev/blob/master/packages/dev/config/jest.js

File: .eslintrc.js (hunk 2 above)
Comment (kirushik) [reply .eslintrc.js:1]: Added codestyle change as a part of #35

File: jest.config.js (hunk 3 above)
Comment (kirushik) [reply jest.config.js:1]: Well, the jest config I have is inherited from my vuejs template; added a note to #34 to revisit this config (and probably borrow the one from polkadot-js) when removing the VueJS

**Comment by Stefie** [comment]
A 100% non-technical feedback: You need to get over the list of words for the recovery phrases: 
The one I got has 3 war references (out of 4 words). It shouldn't contain any violent/ triggering words.

Screenshot 2020-08-05 at 13 03 31

**Comment by kirushik** [comment]
@Stefie thanks for the wordlist feedback, by the way.
I just took the [official bip32 wordlist](https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt) -- but, while your particular outcome is just pure (absence of) luck, I agree that we need to chose better.
I've created #39 to track this; seems to be a trivial issue to fix, hopefully I'll find time to do that shortly.

**Pull request merged**